sandbox/apparmor/profile.go

```diff
@@ -251,3 +286,68 @@ var SnapConfineDistroProfilePath = func() string {
 
 	return ""
 }
+
+// SetupSnapConfineSnippets inspects the system and sets up local apparmor
+// policy for snap-confine. Local policy is included by the system-wide policy.
+// Returns whether any modifications was made to the snap-confine snippets.
+func SetupSnapConfineSnippets() (wasChanged bool, err error) {
+	// Create the local policy directory if it is not there.
+	if err := os.MkdirAll(dirs.SnapConfineAppArmorDir, 0755); err != nil {
+		return false, fmt.Errorf("cannot create snap-confine policy directory: %s", err)
+	}
+
+	// Location of the generated policy.
```

Comment:
this comment wasn't relevant anymore, the glob is not a location??

Reply:
Good point, removed the comment as it had no place there, and wasn't needed after removing the `glob` variable